var/ramble/repos/builtin/applications/gromacs/application.py

```diff
@@ -98,6 +98,49 @@ class Gromacs(SpackApplication):
              input='JCP_benchmarks')
     workload('ion_channel', executables=['pre-process', 'execute-gen'],
              input='JCP_benchmarks')
+    workload('adh_dodec', executables=['pre-process', 'execute-gen'],
+             input='JCP_benchmarks')
+
+    all_workloads = [
+        'water_gmx50',
+        'water_bare',
+        'lignocellulose',
+        'hecbiosim',
+        'benchpep',
+        'benchpep_h',
+        'benchmem',
+        'benchrib',
+        'stmv_rf',
+        'stmv_pme',
+        'rnase_cubic',
+        'ion_channel',
+        'adh_dodec',
+    ]
+
+    workload_variable('gmx', default='gmx_mpi',
+                      description='Name of the gromacs binary',
+                      workloads=all_workloads)
+    workload_variable('grompp', default='{gmx} grompp',
+                      description='How to run grompp',
+                      workloads=all_workloads)
+    workload_variable('mdrun', default='{gmx} mdrun',
+                      description='How to run mdrun',
+                      workloads=all_workloads)
+    workload_variable('nsteps', default=str(20000),
+                      description='Simulation steps',
+                      workloads=all_workloads)
+    workload_variable('resetstep', default='{str(int(0.9*{nsteps}))}',
+                      description='Reset performance counters at this step',
+                      workloads=all_workloads)
+    workload_variable('verbose', default="-v",
```

Comment:
Given verbose mode has a slight performance impact, can we default it to off?

You can even add:
`values=['', '-v']` to tell people how to turn it on.

Reply:
Ok - sounds reasonable. I added valid values to some other variables as well.

For verbose there may be a dependency between the value and what FOMs that are available (if some output  from verbose mode is used for FOMs).